### PR TITLE
Added various TTPS that inhibit system recovery

### PR DESCRIPTION
**Pull request by bfuzzy1**

### CODE REVIEW

File: defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+id: 70a57910-a56a-4b60-bcb4-84b7a56076ab
+name: ' Disable Microsoft Defender firewall'
```
Comment (privateducky) [inline defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml:2]: Remove the single quotes here. Won't cause issues, just consistency.

File: defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  id: T1562.004
+  name: Disable or Modify System Firewall
+platforms:
+  windows:
+    cmd:
+      command: netsh advfirewall set currentprofile state off
+metadata:
+  authors:
+    - bfuzzy1
+    - MITRE
```
Comment (privateducky) [inline defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml:17]: Any of these come from Mitre emulation plans (otherwise don't need them as author)

**Review by privateducky — APPROVED** [review]
These are all great, only a couple minor things. I will merge this next week. Because we're doing deletions in here, I want to put a warning on our impact procedures to ensure users of them understand before running. Great contribution!

File: defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml (hunk 1 above)
Comment (bfuzzy1) [reply defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml:2]: Weird I don't see those on my end.

File: defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml (hunk 2 above)
Comment (bfuzzy1) [reply defense-evasion/70a57910-a56a-4b60-bcb4-84b7a56076ab.yml]: Good to know. I used some TTPs from a previous PT engagement. I went ahead a removed MITRE as the author on what I submitted.

**Comment by bfuzzy1** [comment]
@privateducky good call out I ment to mention that via Slack! 😁

**Comment by privateducky** [comment]
OK merging these in today. We'll apply the destructive tag retroactively, likely during Wednesday's new release.

**Pull request merged**